### PR TITLE
Editting compound types

### PR DESCRIPTION
**Pull request by Tanish-Eagle**
Fixed spelling mistakes, and some grammar mistakes. Also fixed some missing semicolons.

### CODE REVIEW

**Comment by sweet2honey** [comment]
Nice to meet you here! I'm improving the Chinese version recently, and found that your commits offer me quite a hand in revising! THANK YOU and let's make this repo greater LOL :D

**Comment by Tanish-Eagle** [comment]
@sweet2honey Glad to help! This is actually my first open source project. I'm hoping to point to this one day, and say to people with pride that "I worked on that."

**Comment by sunface** [comment]
> @sweet2honey Glad to help! This is actually my first open source project. I'm hoping to point to this one day, and say to people with pride that "I worked on that."

Yes, you will :)

File: en/src/compound-types/string.md [hunk 1]
```diff
@@ -189,8 +189,8 @@ fn main() {
 
     // Byte strings can have byte escapes...
     let escaped = b"\x52\x75\x73\x74 as bytes";
-    // ...but no unicode escapes
-    // let escaped = b"\u{211D} is not allowed";
+    // ...But no unicode escapes
+    // let escaped = b"\u{211D} Is not allowed";
```
Comment (sunface) [inline en/src/compound-types/string.md:193]: Changes here are very nice

File: en/src/compound-types/struct.md [hunk 2]
```diff
@@ -27,35 +27,35 @@ fn main() {
...
+    println!("Success!");
 } 
 
-// fill the blank to make the code work
+// Fill the blank to make the code work
 fn do_something_with_unit(u: __) {   }
 ```
 
-3. 🌟🌟🌟 Tuple struct looks similar to tuples, it has added meaning the struct name provides but has no named fields. It's useful when you want give the whole tuple a name, but don't care the fields's names.
+3. 🌟🌟🌟 Tuple struct looks similar to tuples, it has added meaning the struct name provides but has no named fields. It's useful when you want to give the whole tuple a name, but don't care about the fields's names.
```
Comment (sunface) [inline en/src/compound-types/struct.md:47]: Nice, grammer improving is very helpful for this project, we really need it.

**Review by sunface — APPROVED** [review]
Great work! Improved grammer in this PR is really helpful and important.

**Pull request merged**